connector/src/main/java/org/geysermc/connector/network/session/cache/WorldCache.java

```diff
@@ -38,6 +39,17 @@
     private final GeyserSession session;
     @Setter
     private Difficulty difficulty = Difficulty.EASY;
+
+    /**
+     * True if the client prefers being shown their coordinates, regardless if they're being shown or not.
+     * This will be true everytime the client joins the server because neither the client nor server store the preference permanently.
+     */
+    @Setter
+    private boolean prefersShowCoordinates = true;
+
+    /**
+     * True if the client is being shown their coordinates.
+     */
     private boolean showCoordinates = true;
```

Comment:
This value doesn't appear to be accessed? Just updated.

Reply:
I'll remove it, I suppose if we need that information in the future it can be added